jobbergate-api/tests/apps/job_scripts/test_services.py

```diff
@@ -191,13 +191,38 @@ async def test_delete_updates_related_submissions(
         ]
 
 
+class TestEntryInfo(NamedTuple):
+    """Named tuple to store the info on a test entry."""
+
+    last_updated_delta: int
+    last_used_delta: int | None
+    is_archived: bool
+
+
+def filter_test_entries(**kwargs: set[Any]) -> Callable[[dict["TestEntryInfo", dict[str, Any]]], set[int]]:
+    """
+    Return a function to filter test entries.
```

Comment:
These tests are very convoluted. It would be good to do a little more explaining of what this function is used for.

Reply:
I simplified the function and expanded the docstring for clarification: [0c7f836](https://github.com/omnivector-solutions/jobbergate/pull/563/commits/0c7f8363e6cec4284b7c19c9f3119430c0f2df89)